client.js

```diff
@@ -37,8 +43,10 @@ function createRedirectButton() {
     // bind the Token Button to the Token Controller when ready
     tokenController.bindButtonClick(
         button, // Token Button
-        redirectTokenRequest, // redirect token request function
-        function(error) { // bindComplete callback
+        token.createRequest(() => {
```

Comment:
mmmm this might need additional work, you may have to merge both createRedirectButton and createPopupButton into a single function like we do in merchant-demo, see https://github.com/tokenio/merchant-demo/blob/master/src/components/TokenEnablerButton/index.js#L86

Reply:
done. Made a common function for both.